src/libsystemd/sd-bus/bus-message.c

```diff
@@ -4667,3 +4667,26 @@ _public_ int sd_bus_message_sensitive(sd_bus_message *m) {
         m->sensitive = true;
         return 0;
 }
+
+char** bus_message_make_log_fields(sd_bus_message *m) {
+        _cleanup_strv_free_ char **strv = NULL;
+
+        assert(m);
+
+        (void) strv_extend_join(&strv, "DBUS_MESSAGE_TYPE", "=", bus_message_type_to_string(m->header->type));
+        (void) strv_extend_join(&strv, "DBUS_SENDER", "=", sd_bus_message_get_sender(m));
+        (void) strv_extend_join(&strv, "DBUS_DESTINATION", "=", sd_bus_message_get_destination(m));
+        (void) strv_extend_join(&strv, "DBUS_PATH", "=", sd_bus_message_get_path(m));
+        (void) strv_extend_join(&strv, "DBUS_INTERFACE", "=", sd_bus_message_get_interface(m));
+        (void) strv_extend_join(&strv, "DBUS_MEMBER", "=", sd_bus_message_get_member(m));
```

Comment:
Hmm, would `strv_extend_join` ever be used with something else than "=" as the joiner? It'd be more effiecient to change those calls to
```C
 (void) strv_extend_join(&strv, "DBUS_MESSAGE_TYPE=", bus_message_type_to_string(m->header->type));
...
```